proposals/boost/boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml

```diff
@@ -46,6 +46,11 @@
 				<artifactId>javax.json-api</artifactId>
 				<version>1.1</version>
 			</dependency>
+			<dependency>
+				<groupId>javax.json.bind</groupId>
+				<artifactId>javax.json.bind-api</artifactId>
+				<version>1.0</version>
+			</dependency>
```

Comment:
Just a thought:  did we ever consider pulling in the EE7 / EE8 aggregate dependencies?

Reply:
Same would apply to MP BOMs.